pkg/prometheus/promcfg.go

```diff
@@ -374,6 +374,14 @@ func addSafeTLStoYaml(cfg yaml.MapSlice, namespace string, tls monitoringv1.Safe
 		tlsConfig = append(tlsConfig, yaml.MapItem{Key: "server_name", Value: *tls.ServerName})
 	}
 
+	if tls.MinVersion != "" {
+		tlsConfig = append(tlsConfig, yaml.MapItem{Key: "min_version", Value: tls.MinVersion})
+	}
+
+	if tls.MaxVersion != "" {
+		tlsConfig = append(tlsConfig, yaml.MapItem{Key: "max_version", Value: tls.MaxVersion})
+	}
+
```

Comment:
Do we need to verify the Prometheus version before adding the config? I ask because I'm not sure since when Prometheus import this version of common that adds min/max TLS version 🤔

Reply:
Got it. Thanks

Reply:
Fixed. 
@ArthurSens  @simonpasquier  Please re-check it. thanks